test/common/testlib.py

```diff
@@ -747,7 +747,7 @@ def become_superuser(self, user=None, password=None):
 
         self.open_superuser_dialog()
         self.wait_in_text(".pf-c-modal-box:contains('Switch to administrative access')", f"Password for {user or 'admin'}:")
-        self.set_input_text(".pf-c-modal-box:contains('Switch to administrative access') input", password or "foobar")
+        self.set_input_text(".pf-c-modal-box:contains('Switch to administrative access') input[type=password]", password or "foobar")
```

Comment:
This should be a separate commit (there are more instances of this below).

Reply:
Why separate?  This qualification is now necessary because there are twi `input` elements in the dialog now.

Reply:
Oh, of course -- I misread that.